lib/slax/commands/github_commands.ex

```diff
@@ -27,8 +33,13 @@ defmodule Slax.Commands.GithubCommands do
     end
   end
 
+  @doc """
+  Pulls all issue templates from a story repo that are included in `story_paths`.
+  It  then uses these templates to create issues in a newly created github repository,
+  If there is no github repository the function will exit early
+  """
   def create_reusable_stories(
-        %{project_name: project_name} = results,
+        %{project_name: project_name, github_repo: _} = results,
```

Comment:
requiring this field in the results map allows us to skip the step if there is no `github_repo` to post reusable stories to